terraform/modules/pre-auth-lambda/lambda/lambda_function.py

```diff
@@ -0,0 +1,93 @@
+import os
+import datetime
+from dateutil.relativedelta import relativedelta
+import boto3
+from boto3.dynamodb.conditions import Key
+
+table_name = os.environ.get("TABLE_NAME")
+if table_name is None:
+    message = "Variable TABLE_NAME was not provided."
+    print(message)
+    raise Exception(message)
+region_name = os.environ.get("AWS_REGION")
```

Comment:
possibly a new line to aid in readability

Reply:
This is done.